core/client/app/routes/application.js

```diff
@@ -20,7 +19,7 @@ export default Ember.Route.extend(ApplicationRouteMixin, ShortcutsRoute, {
     notifications: Ember.inject.service(),
 
     afterModel: function (model, transition) {
-        if (this.get('session').isAuthenticated) {
+        if (this.get('session').get('isAuthenticated')) {
```

Comment:
A minor thing but I think we should standardise on `this.get('session.isAuthenticated')` rather than the double get.